tsconfig.json

```diff
@@ -10,7 +10,8 @@
     "allowSyntheticDefaultImports": true,
     "noEmit": false,
     "outDir": "lib",
-    "sourceMap": true
+    "sourceMap": true,
+    "moduleResolution": "nodenext"
```

Comment:
This is required to allow for the typescript-eslint libs to work. See more details - https://github.com/typescript-eslint/typescript-eslint/issues/7284